src/test/java/roomescape/business/service/PlayPlayTimeServiceTest.java

```diff
@@ -0,0 +1,97 @@
+package roomescape.business.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.time.LocalTime;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import roomescape.business.domain.PlayTime;
+import roomescape.fake.FakePlayTimeDao;
+import roomescape.presentation.dto.PlayTimeRequest;
+import roomescape.presentation.dto.PlayTimeResponse;
+
+class PlayPlayTimeServiceTest {
+
+    private static final LocalTime FORMATTED_MAX_LOCAL_TIME = LocalTime.of(23, 59);
+
+    private PlayTimeService playTimeService;
+
+    @BeforeEach
+    void setUp() {
+        playTimeService = new PlayTimeService(new FakePlayTimeDao());
+    }
+
+    @DisplayName("방탈출 시간을 조회한다.")
+    @Test
+    void find() {
+        // given
+        playTimeService.create(new PlayTimeRequest(FORMATTED_MAX_LOCAL_TIME));
+
+        final Long id = 1L;
+        final PlayTime expected = roomescape.business.domain.PlayTime.createWithId(1L, FORMATTED_MAX_LOCAL_TIME);
```

Comment:
불필요한 패키지 경로는 제거해주시면 좋을 것 같아요. 

```suggestion
        final PlayTime expected = PlayTime.createWithId(1L, FORMATTED_MAX_LOCAL_TIME);
```

Reply:
제거하였습니다 !